core/modules/filter/filter.pages.inc

```diff
@@ -296,6 +298,13 @@ function filter_format_editor_image_form($form, &$form_state, $format) {
     '#parents' => array('attributes', 'data-has-caption'),
     '#access' => !empty($format->filters['filter_image_caption']->status),
   );
+  $form['orientate'] = array(
+    '#title' => t('Orientate if necessary'),
+    '#description' => t('If this is not checked, the image orientation will not be altered.'),
+    '#type' => 'checkbox',
+    '#default_value' => (isset($upload_settings['orientate']) ? $upload_settings['orientate'] : FALSE),
+    '#parents' => array('attributes', 'orientate'),
+  );
```

Comment:
Add `#access` here as well based on EXIF being available.

Reply:
I have removed this orientate option because it would require additions to CKEditor plugin script to make it work per upload.